examples/arp_packet.rs

```diff
@@ -51,42 +58,65 @@ fn main() {
     let iface_name = match env::args().nth(1) {
         Some(n) => n,
         None => {
-            writeln!(io::stderr(), "USAGE: packetdump <NETWORK INTERFACE> <SOURCE IP>").unwrap();
+            writeln!(
+                io::stderr(),
+                "USAGE: arp_packet <NETWORK INTERFACE> <SOURCE IP>"
+            ).unwrap();
             process::exit(1);
-        },
+        }
     };
 
     let source_ip: Result<Ipv4Addr, AddrParseError> = match env::args().nth(2) {
         Some(n) => n.parse(),
         None => {
-            writeln!(io::stderr(), "USAGE: packetdump <NETWORK INTERFACE> <SOURCE IP> <TARGET IP> <TARGET MAC>").unwrap();
+            writeln!(
+                io::stderr(),
+                "USAGE: arp_packet <NETWORK INTERFACE> <SOURCE IP> <TARGET IP> <TARGET MAC>"
+            ).unwrap();
             process::exit(1);
-        },
+        }
     };
 
     let target_ip: Result<Ipv4Addr, AddrParseError> = match env::args().nth(3) {
         Some(n) => n.parse(),
         None => {
-            writeln!(io::stderr(), "USAGE: packetdump <NETWORK INTERFACE> <SOURCE IP> <TARGET IP> <TARGET MAC>").unwrap();
+            writeln!(
+                io::stderr(),
+                "USAGE: arp_packet <NETWORK INTERFACE> <SOURCE IP> <TARGET IP> <TARGET MAC>"
+            ).unwrap();
             process::exit(1);
-        },
+        }
     };
 
     let target_mac: Result<MacAddr, ParseMacAddrErr> = match env::args().nth(4) {
         Some(n) => n.parse(),
         None => {
-            writeln!(io::stderr(), "USAGE: packetdump <NETWORK INTERFACE> <SOURCE IP> <TARGET IP> <TARGET MAC>").unwrap();
+            writeln!(
+                io::stderr(),
```

Comment:
There is an `eprintln!` macro stabilized now. However it does not work for the oldest Rust we say we support with libpnet. So to use that we would need to bump the minimum required version to 1.19. Not saying we need to do that. Just leaving that comment.